src/state/useAppState.ts

```diff
@@ -49,36 +49,29 @@ export type AppState = {
   }[];
 };
 
+function orZero(num: number): number {
+  return num < 0 ? 0 : num;
+}
+
 export default function useAppState(): [
   AppState,
   Dispatch<Action>,
   () => ClosedRound,
+  () => GremlinId | null,
 ] {
   const [state, dispatch] = useReducer(gameReducer, INITIAL_STATE);
 
-  const effects = getAllRoundEffects(state.pastRounds);
-  const roundCapacity = getCapacity(effects);
+  const effects = getAllEffects(state);
 
-  const finishedActionIds = concatByProp(
-    state.pastRounds,
-    'selectedGameActionIds',
-  );
-  const thisRoundsActionUserStoryEffects = state.currentRound.selectedGameActionIds
-    .map((id) => getEffect(id, 0, finishedActionIds))
-    .filter(isEffect)
-    .filter(isUserStoryChanceEffect);
+  const roundCapacity = orZero(getCapacity(effects));
```

Comment:
found a bug where capacity went < 0 and crashed the game by having all gremlins + drag effects active.